src/node.cc

```diff
@@ -4244,11 +4249,15 @@ static void StartNodeInstance(void* arg) {
       SealHandleScope seal(isolate);
       bool more;
       do {
+#ifndef NODE_EMBEDDED_MODE
         v8::platform::PumpMessageLoop(default_platform, isolate);
+#endif  // NODE_EMBEDDED_MODE
         more = uv_run(env->event_loop(), UV_RUN_ONCE);
 
         if (more == false) {
+#ifndef NODE_EMBEDDED_MODE
           v8::platform::PumpMessageLoop(default_platform, isolate);
```

Comment:
In Electron, the message loop is managed by Electron itself, so the embedder would be responsible for performing the tasks. In practice `StartNodeInstance` is never used by embedders. This line is commented out because for embedders `v8_libplatform` is not compiled by default.